media-sound/jack/jack-4.0.0_pre20200706.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
+PYTHON_REQ_USE="ncurses"
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
```

Comment:
Not actually tested but we may need DISTUTILS_USE_SETUPTOOLS=rdepend because of [this](https://github.com/jack-cli-cd-ripper/jack/blob/python3-mb/setup.py#L27) (see [here](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#different-build-system-variations)).

Reply:
done

Reply:
Thanks!